.eslintrc.json

```diff
@@ -1,11 +1,12 @@
 {
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
   "extends": [
     "@skyscanner/eslint-config-skyscanner",
-    "plugin:flowtype/recommended"
+    "plugin:ft-flow/recommended"
```

Comment:
Successor - https://www.npmjs.com/package/eslint-plugin-ft-flow

Reply:
We can remove this one as we don't use flow in this repo anymore